tests/functional/u/used/used_before_assignment_typing.py

```diff
@@ -7,11 +7,12 @@
 if TYPE_CHECKING:
     if True:  # pylint: disable=using-constant-test
         import math
+    import dbm
+    print(dbm)
```

Comment:
May I ask what this line tests?

Reply:
I believe this was discussed in previous review: https://github.com/pylint-dev/pylint/pull/8441/files/82fdb4e39cd4716a5b5a080f3fa3f56a3ffb1f03#diff-1c142afc02e8b8c4cf872f61ff8693a0b25646d6db50f16f8027dc130e7f719f.

But the test is to ensure that no warning is emitted when a variable is defined and used in an always false branch.

Reply:
Thanks, sorry for the repeat question. At your option you could add a brief comment to this effect, but no worries.